chainer/training/triggers/interval.py

```diff
@@ -40,9 +42,16 @@ def __call__(self, trainer):
         """
         updater = trainer.updater
         if self.unit == 'epoch':
-            prev = self.count
-            self.count = updater.epoch_detail // self.period
-            return prev != self.count
+            epoch_detail = updater.epoch_detail
+            previous_epoch_detail = updater.previous_epoch_detail
```

Comment:
> However, if the iterator/updater does not have previous_epoch_detail, there is a risk of resuming issue (#2244). 
I see. Thank you for making the problem clearer. I will ask other core members how to deal with non-existence of `previous_epoch_detail`. Aside from that, showing a warning message seems a good idea to me.